File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/swift/ADV-13.swift
import Foundation
struct A { var v: Int64 }
struct B { var v: Int64 }
func main() {
    print("ADV-START"); fflush(stdout)
    let a = A(v: 42)
    let bridge: Any = a
    let b = bridge as! B
    print("OBS=V:\(b.v)"); fflush(stdout)
    print("ADV-END"); fflush(stdout)
}
main()

File: benchmark/template/programs/kotlin/adversarial/ADV-13.kt
class A(val v: Long)
class B(val v: Long)

fun main() {
    println("ADV-START")
    System.out.flush()
    val a: Any = A(42L)
    val b: B = a as B
    println("OBS=V:${b.v}")
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}
